packages/wonder-blocks-button/components/button-core.js

```diff
@@ -0,0 +1,271 @@
+// @flow
+import React from "react";
+import {StyleSheet, css} from "aphrodite";
+
+import {LabelLarge} from "wonder-blocks-typography";
+import {processStyleList} from "../../wonder-blocks-core/util/util.js";
+import Color, {mix, fade} from "wonder-blocks-color";
+import type {ValidTints} from "wonder-blocks-color";
+import type {SharedProps} from "../types.js";
+
+type Props = SharedProps & {
+    hovered?: boolean,
+    focused?: boolean,
+    pressed?: boolean,
+};
+
+export default class ButtonCore extends React.Component<Props> {
+    render() {
+        const {
+            color,
+            kind,
+            light,
+            size,
+            disabled,
+            hovered,
+            focused,
+            pressed,
+        } = this.props;
+
+        const buttonStyles = _generateStyles(
+            color || Color.blue,
+            kind || "primary",
+            !!light,
+        );
+
+        if (disabled) {
```

Comment:
Hmm - it doesn't feel like we need a whole separate branch for the disabled setting. I think we can just do:

```
disabled && sharedStyles.disabled
...
disabled && buttonStyles.disabled
```

Reply:
It should be fine to do `disabled={disabled}` on the button.

Reply:
Ohhh did not know you could do that. Thanks!